gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/RequesterService.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gobblin.service;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.type.TypeReference;
+
+import com.google.common.collect.Lists;
+import com.linkedin.restli.server.resources.BaseResource;
+
+import edu.umd.cs.findbugs.annotations.SuppressWarnings;
+
+
+/**
+ * Use this class to get who sends the request.
+ */
+public abstract class RequesterService {
+
+  public static final String REQUESTER_LIST = "gobblin.service.requester.list";
+
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+
+  // user should provide customized logic to get real requester
+  @SuppressWarnings(value = "MS_SHOULD_BE_FINAL",
+      justification = "This is not final variable because derived service can override it.")
+  public static RequesterService rs = new NoopRequesterService();
```

Comment:
Please try injection for this so that it is not tied to a single global instance. Currently we don't have a use for multiple instances, but I think the injection will be cleaner.